kubernetes_state/metadata.csv

```diff
@@ -29,6 +29,7 @@ kubernetes_state.endpoint.address_not_ready,gauge,,,,Number of addresses not rea
 kubernetes_state.endpoint.created,gauge,,,,Unix creation timestamp,0,kubernetes,k8s_state.endpoint.creation_time
 kubernetes_state.job.status.failed,counter,,,,Observed number of failed pods in a job,0,kubernetes,k8s_state.job.failed
 kubernetes_state.job.status.succeeded,counter,,,,Observed number of succeeded pods in a job,0,kubernetes,k8s_state.job.succeeded
+kuberentes_state.job.duration_time,gauge,,,,Job duration time,0,kubernetes,k8s_state.job.duration_time
```

Comment:
```suggestion
kuberentes_state.job.duration_time,gauge,,second,,Job duration time,0,kubernetes,k8s_state.job.duration_time
```
The unit is missing. Not sure if that's in second or millisecond or nanosecond..

Reply:
Thank you! The unit is correct in seconds because it is a subtraction of unixtimestamp.